add-chain/main.go

```diff
@@ -227,3 +258,25 @@ func getGasPayingToken(l1rpcURl string, SystemConfigAddress superchain.Address)
 
 	return (*superchain.Address)(&result.Addr), nil
 }
+
+func copyDeployConfigFile(chainId uint64, sourcePath string, targetDir string) error {
+	data, err := os.ReadFile(sourcePath)
+	if err != nil {
+		return err
+	}
+	return os.WriteFile(path.Join(targetDir, "deploy-config.json"), data, 0o777)
+}
+
+func writeGenesisValidationMetadata(chainId uint64, commit string, targetDir string) error {
+	vm := genesis.ValidationMetadata{
+		GenesisCreationCommit:  commit,
+		NodeVersion:            "18.12.1",
+		MonorepoBuildCommand:   "pnpm",
+		GenesisCreationCommand: "opnode1",
```

Comment:
For now these will be hardcoded. If auto validation fails, these fields can be manually modified to try and get it to pass.

Reply:
Perhaps declare them as globals with a comment, so it'll catch attention easily?